vhdbuilder/packer/pre-install-dependencies.sh

```diff
@@ -120,7 +120,7 @@ if [[ ${OS} == ${MARINER_OS_NAME} ]] && [[ "${ENABLE_CGROUPV2,,}" == "true" ]];
   enableCgroupV2forAzureLinux
 fi
 
-if [[ "${UBUNTU_RELEASE}" == "22.04" ]]; then
+if [[ "${UBUNTU_RELEASE}" == "22.04" && "${ENABLE_FIPS,,}" != "true" ]]; then
```

Comment:
nit `== "false"`

Reply:
that won't work, most non-fips VHD build don't explicitly pass in "Enable_FIPS = false, ", but just without this variable.

Reply:
fair enough... though I think we always make sure to set it when adding new defs

Reply:
ah yeah I do see it's missing from some defs